eclipse-collections-api/src/main/java/org/eclipse/collections/api/multimap/Multimap.java

```diff
@@ -455,6 +455,36 @@
      */
     <K2, V2, R extends MutableMultimap<K2, V2>> R collectKeysValues(Function2<? super K, ? super V, Pair<K2, V2>> function, R target);
 
+    /**
+     * Returns a new multimap with the results of applying the specified keyFunction and valueFunction on each key and corresponding values of the source multimap.
+     * This method is also commonly called transform or map.
+     * <p>
+     * <pre>e.g.
+     * return multimap.collectKeyMultiValues(each -&gt; each + 1, Person::getLastName);
+     * </pre>
+     *
+     * @param keyFunction   {@link Function} to use transformation to get the key
+     * @param valueFunction {@link Function} to use transformation to get the values
+     * @return a new {@code Multimap}, which contains elements as a result of the transformation
+     * @since 10.0
+     */
+    <K2, V2> Multimap<K2, V2> collectKeyMultiValues(Function<? super K, ? extends K2> keyFunction, Function<? super V, ? extends V2> valueFunction);
+
+    /**
+     * Same as the collectKeyMultiValues method but uses the specified target multimap for the results.
+     * <p>
+     * <pre>e.g.
```

Comment:
Both "e.g."s should be outside the pre-formatted sections

Reply:
It’s consistent with rest of the file and the build is going through fine also when I ctrl+q in IntelliJ the javadoc looks fine

Reply:
I'm not saying the Javadoc will break, I'm saying it's conceptually wrong. It's formatted as code, but it's part of the prose.

Reply:
I can have another commit fixing all this. But I don’t think that is critical right now.

Reply:
I can have another commit fixing all this. But I don’t think that is critical right now.